int32 ROOM_ID
string ROOM_NAME
float32 BATTERY_LEVEL
